src/pixee-platform.ts

```diff
@@ -9,8 +9,9 @@ export async function uploadInputFile(tool: TOOL_PATH, file: string) {
   const fileContent = fs.readFileSync(file, "utf-8");
   const form = new FormData();
   form.append("file", fileContent);
+  const pixeeUrl = core.getInput("pixee-api-url");
 
-  const token = await core.getIDToken(AUDIENCE);
+  const token = await core.getIDToken(pixeeUrl);
```

Comment:
You need to keep the audience since that is `https://app.pixee.ai` not `api.pixee.api`

Reply:
Yes, I believe that's true, but it does beg the question, _is https://app.pixee.ai always the right audience_? Maybe it's technically incorrect in some deployments but still good enough? @ryandens please opine.

Reply:
It captures the intended purpose (ensuring the person with permissions to generate this ID token generated it for Pixee). Changing the audience verification server side would be pretty trivial, but would require additional configuration when deploying a pixee server. I don't see a reason to not make this configurable now.


I also don't think we have a great reason for making the `audience` app.pixee.ai when the action uploads to `api.pixee.ai` - i think I picked the audience before we had the `CNAME` for this API. Open to `pixee.ai` or `api.pixee.ai` to match the target destination

Reply:
Any reason to configure them separately, or should we derive the audience from the URL?

Reply:
IMO we should derive the audience, that way one less input they need to pass. But I think this should be done in a different PR to avoid rejecting requests from the actions, since we need to update the server side validation. 

Maybe we should first update the platform to check the audience to see if it is the old or the new api.pixee.ai value, after that change is deploy we can update the logic in the action to derive it from the URL. Finally, we can update the platform again to use something more configurable so it cover the pixee server case

Reply:
That sounds good to me.